### PR TITLE
fixes loris links in theme

### PR DESCRIPTION
**Pull request by sdellis**

### CODE REVIEW

File: sites/all/themes/pul_versailles/css/pul-versailles.no-query.css [hunk 1]
```diff
@@ -257,7 +257,7 @@ a.popup:link {
   padding: 0;
   background-size: 1600px auto;
   max-height: 200px;
-  background: url(https//libimages1.princeton.edu/loris/pudl0083/8206793/00000014.jp2/1212,2400,4736,600/1600,/0/default.jpg) bottom no-repeat;
+  background: url(https//libimages.princeton.edu/loris/pudl0083/8206793/00000014.jp2/1212,2400,4736,600/1600,/0/default.jpg) bottom no-repeat;
```
Comment (kevinreiss) [inline sites/all/themes/pul_versailles/css/pul-versailles.no-query.css]: donesn't this one need a ":".

Comment (sdellis) [inline sites/all/themes/pul_versailles/css/pul-versailles.no-query.css]: nice catch...

**Comment by sdellis** [comment]
@kevinreiss I updated this

**Comment by jpstroop** [comment]
@sdellis is this urgent? Happy to merge.

**Comment by sdellis** [comment]
@jpstroop Not urgent. It's just the banner... go ahead and merge and I'll deploy it when I get a chance.

**Comment by jpstroop** [comment]
👍

**Pull request merged**